src/ts/browserutils.ts

```diff
@@ -25,6 +25,13 @@ export class BrowserUtils {
     return navigator && navigator.userAgent && /Android/.test(navigator.userAgent);
   }
 
+  static get isIOS(): boolean {
+    if (!this.windowExists()) {
+      return false;
+    }
+    return navigator && navigator.userAgent && /iPad|iPhone|iPod/.test(navigator.userAgent);
```

Comment:
We have one more problem:
- iPadOS (iOS 13 on an iPad) will be recognized as `macOS` as they don't have `iPad` in the `userAgent`. We should also check for those devices.

I think we will need to make the check like: 
`isIos` || (`isMacOs` && `isMobile`) you could check for the `touch` events. Or research quickly if there is already a way to detect this.